web/templates/popovers/user_group_info_popover.hbs

```diff
@@ -26,23 +26,59 @@
         <li role="separator" class="popover-menu-separator"></li>
         <li role="none" class="popover-menu-list-item">
             {{#if (or members.length subgroups.length)}}
-                <ul class="popover-menu-list popover-group-menu-member-list" data-simplebar data-simplebar-tab-index="-1" data-simplebar-auto-hide="false">
-                    {{#each subgroups}}
+                <ul class="popover-menu-list popover-group-menu-member-list" data-simplebar
+                  data-simplebar-tab-index="-1" data-simplebar-auto-hide="false">
+                    {{#each displayed_subgroups}}
                         <li class="popover-group-menu-member">
-                            <i class="popover-group-member-icon popover-menu-icon zulip-icon zulip-icon-triple-users" aria-hidden="true"></i>
+                            <i class="popover-group-member-icon popover-menu-icon zulip-icon zulip-icon-triple-users"
+                              aria-hidden="true"></i>
                             <span class="popover-group-menu-member-name">{{name}}</span>
                         </li>
                     {{/each}}
-                    {{#each members}}
+                    {{#each displayed_members}}
                         <li class="popover-group-menu-member">
                             {{#if is_bot}}
-                                <i class="popover-group-member-icon zulip-icon zulip-icon-bot" aria-hidden="true"></i>
+                                <i class="popover-group-member-icon zulip-icon zulip-icon-bot"
+                                  aria-hidden="true"></i>
                             {{else}}
-                                <span class="popover-group-member-icon user-circle zulip-icon zulip-icon-{{user_circle_class}} {{user_circle_class}} popover-group-menu-user-presence hidden-for-spectators" data-tippy-content="{{user_last_seen_time_status}}"></span>
+                                <span
+                                  class="popover-group-member-icon user-circle zulip-icon zulip-icon-{{user_circle_class}} {{user_circle_class}} popover-group-menu-user-presence hidden-for-spectators"
+                                  data-tippy-content="{{user_last_seen_time_status}}"></span>
                             {{/if}}
                             <span class="popover-group-menu-member-name">{{full_name}}</span>
                         </li>
                     {{/each}}
+                    {{#unless display_all_subgroups_and_members}}
+                        {{#if is_system_group}}
+                            {{#if has_bots}}
+                                <li class="popover-group-menu-member">
+                                    <span class="popover-group-menu-member-name">
```

Comment:
Can we move this `li` and `span` element outside the if and just have the text and links element inside the if-else block.